src/gsstunnel.h

```diff
@@ -3,8 +3,9 @@
 #include <stdbool.h>
 #include <ini_configobj.h>
 
-#define MAX_MSG_SIZE 1024*1024 /* 1 Mib */
+#define MAX_MSG_SIZE 16*1024 /* 16 Kib */
 #define AUTOCLEAN(def, fn) def __attribute__ ((__cleanup__(fn)))
+#define PKT_GSSENCREQ   80877104
```

Comment:
what is this?
Looks like the second part of the buffer you put up there, what does it represent?

Is there a spec for this format?